docs/hotkeys.md

```diff
@@ -83,16 +83,16 @@
 |Autocomplete @mentions, #stream_names, :emoji: and topics|<kbd>Ctrl</kbd> + <kbd>f</kbd>|
```

Comment:
Thanks for finding these!

I understand what this means, but for future readers of the log, it would be useful to clarify in what way these are missing, in the commit text.

For example, where did you find this information?

Reply:
It seems this is resolved? 0d1babd now explains that they keys are coming from `urwid_readline`.